### DISCUSSION TITLE
Reordering feeds

### DISCUSSION THREAD

**Discussion by omniraptorr** [pdelfan/ouranos#8] [Ideas]
The default bsky app lets you reorder the feeds that show up in the top navbar and feed list. However, they only let you reorder them by buttons that shift each feed one space at a time instead of a modern drag and drop interface. We can leapfrog them into the 21st century of ux design.

(Or we could just get the order from the main bsky app, but idk if that's exposed anywhere)

**Comment by pdelfan** [comment]
This is on my to-do list, thanks for reminding me!
